test/protocols/test_bad_interface_clamp_and_init.txt

```diff
@@ -0,0 +1,19 @@
+# Specifies an invalid interface: clamped variable can't have an initial value
+
+namespace oxmeta = "https://chaste.comlab.ox.ac.uk/cellml/ns/oxford-metadata#"
+
+units {
+    V = volt
+}
+
+model interface {
+    input oxmeta:membrane_voltage units V = 1
+    clamp oxmeta:membrane_voltage to 2 :: V
```

Comment:
This form is just syntactic sugar for the `define` case above. You want just `clamp oxmeta:membrane_voltage` to have something appear in `self._clamps`. And in that case I think it would be OK to have an initial value on the input.

Reply:
Oh dear. So we'd have to catch duplicates at the parse level then?

Reply:
Well, if we wanted to know what the user did... Less-specific error message if we catch it later

Reply:
Less-specific error is fine for now - it would appear as a straightforward over-defined variable, and should be easy for them to find.